### ISSUE TITLE
Get actual file name

### ISSUE THREAD

**Issue by chintan-mehta24** [MathijsHaakman/cordova-filechooser#7]
I don't think this is a filechooser issue itself (thanks for the plugin by the way!) but I am struggling with copying a file to the app directory using the file chooser and keeping it's file name.

For example, I've got a file called PC-Plus.doc in my Downloads folder, when I select it with the file chooser, I get:

content://com.android.providers.downloads.documents/document/{hashcode}

and with this information I've not been able to retrieve the actual file name, filetype!

Are you able to point my question?

**Comment by MathijsHaakman** [comment]
Yeah, this point has been brought up before in the original repository. I'll refer you to these issues.

- [https://github.com/don/cordova-filechooser/issues/13](Get the real file path from a content:// filepath)
- [https://github.com/don/cordova-filechooser/issues/6](Get the actual file name from a file)

I'll hope this will answers your questions
